Dockerfile.prod

```diff
@@ -1,51 +1,66 @@
-FROM node:20-alpine3.18 as build
-# Installing libvips-dev for sharp Compatibility
-RUN apk update && apk add --no-cache git libc6-compat build-base gcc bash autoconf automake zlib-dev libpng-dev vips-dev > /dev/null 2>&1
+FROM node:20-alpine3.18 AS build
+# Update package index and install build dependencies.
+# These packages are needed for building native modules and ensuring compatibility.
+RUN apk update && apk add --no-cache \
+    git libc6-compat build-base gcc bash autoconf automake \
+    zlib-dev libpng-dev vips-dev shadow python3 make g++ && \
+    # Create a symlink for python3 so node-gyp can locate Python as 'python'
+    ln -sf $(which python3) /usr/bin/python && \
+    echo "Python version: $(python --version)"
 
-ARG NODE_ENV=production
```

Comment:
Ik zie deze `NODE_ENV=production` niet meer terug in de nieuwe versie. Klopt het dat deze verwijderd wordt?